.github/workflows/build-wheel-linux-x86_64.yaml

```
@@ -222,11 +226,15 @@ jobs:
     - name: Checkout Catalyst repo
       uses: actions/checkout@v3
 
-    - name: Install dependencies (CentOS)
+    - name: Install dependencies (AlmaLinux)
       run: |
         # Reduce wait time for repos not responding
-        cat /etc/yum.conf | sed "s/\[main\]/\[main\]\ntimeout=5/g" > /etc/yum.conf
-        yum update -y && yum install -y cmake ninja-build openmpi-devel libzstd-devel
+        cat /etc/dnf.conf | sed "s/\[main\]/\[main\]\ntimeout=5/g" > /etc/dnf.conf
+        dnf update -y && dnf install -y cmake ninja-build openmpi-devel libzstd-devel
+        dnf clean all -y
+        dnf install gcc-toolset-13 -y
+        source ls /opt/rh/gcc-toolset-13/enable -y
```

```suggestion
        source /opt/rh/gcc-toolset-13/enable -y
```

Though, I think the above just create a new bash session that goes away at the end of the run step. To ensure this is always used I think something like
```
"{path}" >> $GITHUB_PATH
```
is required, as it will persist across the rest of the steps (https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#adding-a-system-path). If other paths are needed, maybe something like https://github.com/orgs/community/discussions/25267#discussioncomment-3247178 but validating they are run inside the container before hitting the build steps may help.

---

- Done with the code suggestion.
- Using echo "/some/path" >> $GITHUB_PATH did not seem to work. So, the PATH and LIBRARY_PATH environment variables have to be set inside every task that needed them.